app/views/orgs/roster_entries/assignment_repos/_linked_accepted.html.erb

```diff
@@ -10,7 +10,11 @@
     <div class='d-flex flex-column flex-justify-around'>
       <div>
         <%= link_to view.github_user_url, class: "assignment-repo-github-url", target: '_blank' do %>
-          <h3 class="css-truncate css-truncate-target"><%= view.github_user_login %></h3>
+          <% if defined? current_roster_entry %>
```

Comment:
Can we bring this logic out into the view model? The method could be something like `view.user_display_name` or something similar.

Reply:
I looked into it and it's going to be a bit tricky to do that. Because `current_roster_entry` is not always defined and it's going to be hard to pass that to the view model. We get an error that `current_roster_entry` is not defined. The error happens at the view level. So we will still need an `if` statement.

Reply:
Oh I see. This is fine :+1: